### PR TITLE
Update `README.rst` with new events and requirements

### PR DESCRIPTION
**Pull request by cmagnobarbosa**
# Pull Request Template

### :octocat: Are you working on some issue? Identify the issue!

https://github.com/kytos/kytos/issues/1225
### :bookmark_tabs: Description of the Change

Update `README.rst` file  with new events and requirements.

### :page_facing_up: Release Notes

- N/A

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -58,6 +58,7 @@ Requirements
 ############
 
 - kytos/of_core
+- Kytos/flow_manager
```
Comment (josemauro) [inline README.rst]: You are starting Kytos/flow_manager with upper case "K" here and the next lines that use the "kytos/napp" notation.

**Review by josemauro — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (cmagnobarbosa) [reply README.rst]: Resolved.

File: README.rst [hunk 2]
```diff
@@ -79,6 +80,34 @@ Content
...
+.. code-block:: python3
+
+   {
+     'dpid': <switch.id>
+   }
+
+kytos/topology.switch.disabled
+==============================
+Listen when a switch was disabled. This event is used to indicate when to
+remove LLDP flows through Kytos/flow_manager.
```
Comment (hdiogenes) [inline README.rst]:
```suggestion
remove LLDP flows through kytos/flow_manager.
```

File: README.rst [hunk 3]
```diff
@@ -79,6 +80,34 @@ Content
       'source': <object>, # instance of kytos.core.switch.Connection class
     }
 
+kytos/topology.switch.enabled
+=============================
+Listen when a switch was enabled. This event is used to indicate when to
+install LLDP flows through Kytos/flow_manager.
```
Comment (hdiogenes) [inline README.rst]:
```suggestion
install LLDP flows through kytos/flow_manager.
```

**Review by hdiogenes — APPROVED** [review]

**Pull request merged**